tst/rref/is_rref.tst

```diff
@@ -0,0 +1,4 @@
+gap> ReadPackage("GaussPar", "gap/RREF.g");;
```

Comment:
Please move this file into `tst/standard/`. I'ld like `standard/` to contain all sequential tests for functionality of this package, and `parallel/` should contain the tests for the code paths where we compute in parallel. The content of `stats/` can stay in its own folder, since there we test only our benchmarking suite.